src/markdown_generators.js

```diff
@@ -4,35 +4,66 @@ const yaml = require("js-yaml")
 const markdown = require("markdown-builder")
 const titleCase = require("title-case")
 const TurndownService = require("turndown")
+const turndownPluginGfm = require("turndown-plugin-gfm")
+const helpers = require("./helpers")
+const gfm = turndownPluginGfm.gfm
+const tables = turndownPluginGfm.tables
 const turndownService = new TurndownService()
+turndownService.use(gfm)
+turndownService.use(tables)
+
+const REPLACETHISWITHAPIPE = "REPLACETHISWITHAPIPE"
+const REFSHORTCODESTART = "REFSHORTCODESTART"
+const REFSHORTCODEEND = "REFSHORTCODEEND"
+
+/**
+ * Sanitize markdown table content
+ **/
 turndownService.addRule("table", {
   filter:      ["table"],
-  replacement: content => {
-    return `{{< rawhtml >}}<table>${content}</table>{{< /rawhtml >}}`
-  }
-})
-turndownService.addRule("th", {
-  filter:      ["th"],
-  replacement: content => {
-    return `<th>${content}</th>`
-  }
-})
-turndownService.addRule("tr", {
-  filter:      ["tr"],
-  replacement: content => {
-    return `<tr>${content}</tr>`
-  }
-})
-turndownService.addRule("td", {
-  filter:      ["td"],
-  replacement: content => {
-    return `<td>${content}</td>`
+  replacement: (content, node, options) => {
+    /**
+     * Interate the HTML node and replace all pipes inside table
+     * cells with a marker we'll use later
+     */
+    for (let i = 0; i < node.rows.length; i++) {
+      const cells = node.rows[i].cells
+      for (let j = 0; j < cells.length; j++) {
+        cells[j].innerHTML = cells[j].innerHTML.replace(
+          /\|/g,
+          REPLACETHISWITHAPIPE
+        )
+      }
+    }
```

Comment:
I avoided doing it this way specifically because the "node" object passed into this function is a native DOM HTMLElement.  The code actually breaks if I put in what you have above, because forEach isn't a function on `.rows` or `.cells`.

Reply:
ahh yeah on HTMLElement you have to do something like [...el].forEach to be able to use it.. tbh I don't know why .forEach isn't implemented there...kind of annoying.